simulation/adx_gyro_accelerometer/src/main/native/cpp/ADXRS450_SpiGyroWrapperData.cpp

```diff
@@ -61,13 +126,16 @@ void ADXRS450_SpiGyroWrapper::CancelAngleCallback(int32_t uid) {
 void ADXRS450_SpiGyroWrapper::InvokeAngleCallback(HAL_Value value) {
   InvokeCallback(m_angleCallbacks, "Angle", &value);
 }
-double ADXRS450_SpiGyroWrapper::GetAngle() { return m_angle; }
+double ADXRS450_SpiGyroWrapper::GetAngle() {
```

Comment:
Are there not empty lines between the function definitions? There should be.

Reply:
Formatter doesn't catch it, added them manually

Reply:
It isn't enforced by wpiformat yet because I haven't merged the styleguide PR that handles it.